### ISSUE TITLE
somenthing trobles me when i am running the testchain locally

### ISSUE THREAD

**Issue by wangyinglike1234** [makerdao/testchain#28]
I am studying the testchain and trying to run the program locally by the github address:
https://github.com/makerdao/developerguides/blob/master/devtools/test-chain-guide/test-chain-guide.md.
However,the github address tells me to export an address called MCD_MOM, but i don't find the address in the file :https://github.com/makerdao/testchain/blob/dai.js/out/addresses-mcd.json.I 'm not sure but i suggest it may because not updated,right? And i want to know the use of the cat and flipper contract, how should i do?

**Comment by wangyinglike1234** [comment]
@savil

**Comment by savil** [comment]
@wangyinglike1234 I don't work actively on makerdao. Its probably better for someone from the team to comment (cc @levity), or you can ask in https://chat.makerdao.com

It seems like the MCD_MOM contract was removed a long time ago (in march/april) when moving from version 0.2.2 to 0.2.3 of the Kovan testnet release (click on those versions in https://changelog.makerdao.com and then click on "contract addresses").

Looking into the git history of `addresses-mcd.json` seems like it was removed in this change (https://github.com/makerdao/testchain/commit/8875cfd78c69d1332ec6bad368aee326d64e4d72#diff-be16e876619379cd2c1f06f17a0978c3), and so maybe MCD_MOM was renamed to MCD_PAUSE? Just guessing. Better to ask in the chat, or @levity.


> And i want to know the use of the cat and flipper contract, how should i do?

you'll need to be more specific in your question. What are you trying to do?

**Comment by godsflaw** [comment]
Correct, there is not currently an `MCD_MOM` and it was replaced with `MCD_PAUSE` a.k.a. the Governance Security Module (GSM).  We are likely to introduce other `MOM` contracts in the future, especially for bypassing the GSM for a select number of calls where the delay harms us more than helps us, but for now there are no `MOM`s.

**Comment by wangyinglike1234** [comment]
@savil @godsflaw thanks for your answer.
Now i am trying to test the mechanism of the mcd's auction，but i don't find the url .The only url like https://mcd-cdp-portal-git-develop.mkr-js-prod.now.sh/?network=kovan i can only trade ,save and borrow DAI .How can i test the flip.sol auction.Looking forward to your answers
@levity

**Comment by levity** [comment]
We don't have a UI for auctions at this time; you can work with them through interacting with the smart contracts directly. Please join https://chat.makerdao.com where many more people can help you with these questions.

**Comment by wangyinglike1234** [comment]
@levity 
Thanks for your answer,questions about the testchain i will go to the chat room for further help.
I also have some other questions .Whether the 'bite' function will be open to us in the future or the 'bite' funtion only be executed officially? And how can I do  auctions now?If the users cannot do auctions how can the MCD system keep the system stable?
